client/electron/package.json

```diff
@@ -22,11 +22,12 @@
     },
     "dependencies": {
         "@capacitor-community/electron": "^5.0.1",
+        "@sentry/electron": "^5.2.0",
         "chokidar": "~3.6.0",
         "electron-log": "^5.1.2",
         "electron-serve": "~1.3.0",
         "electron-unhandled": "~4.0.1",
-        "electron-updater": "~6.1.8",
+        "electron-updater": "6.1.8",
```

Comment:
Not related to the PR but `electron-updater` was updated when adding `sentry/electron` and there seem to be a breaking change with `6.1.9`